protobuf/dkg/dkg.pb.go

```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.32.0
-// 	protoc        v4.25.3
+// 	protoc        v3.20.3
```

Comment:
an unfortunate version mismatch that changed much more files than need be

Reply:
ah fuck